src/xtopcom/xdata/xelect_transaction.hpp

```diff
@@ -98,11 +98,14 @@ class xtcc_transaction_t : public xserializable_based_on<void> {
         XADD_ONCHAIN_GOVERNANCE_PARAMETER(tx_deposit_gas_exchange_ratio);
         XADD_ONCHAIN_GOVERNANCE_PARAMETER(cpu_gas_exchange_ratio);
         XADD_ONCHAIN_GOVERNANCE_PARAMETER(usedgas_decay_cycle);
+        XADD_ONCHAIN_GOVERNANCE_PARAMETER(usedgas_reset_interval); // new name of 'usedgas_decay_cycle'
```

Comment:
should add #if defined(XCHAIN_FORKED_BY_DEFAULT)， otherwise here will modify tcc contract‘s genesis block’